packages/cacao/package.json

```diff
@@ -48,7 +48,8 @@
     "@ipld/dag-cbor": "^9.0.6",
     "caip": "^1.1.0",
     "multiformats": "^12.1.3",
-    "uint8arrays": "^4.0.9"
+    "uint8arrays": "^4.0.9",
+    "viem": "^1.19.13"
```

Comment:
Any way to avoid importing this full library for just one utility function please? Maybe there are alternative packages that would be lighter?

Reply:
Viem is very likely going to be a dependency for the upcoming [Varsig implementation](https://github.com/ceramicnetwork/js-did/blob/feat/varsig/packages/varsig/package.json#L66), which means it's pretty much always going to be a requirement everywhere. 

I think viem is pretty tree-shakable, so it shouldn't have much of an impact on actual build sizes?

Reply:
OK thanks, sounds good if tree-shaking works well 👍

Reply:
Well that's what I see on https://bundlephobia.com/package/viem

Can I trust the badge shown there?

Reply:
I don't know how reliable it is, I guess it depends on our own build process as well?
Would be good to figure out though, the lib is 200kB+ so a pretty big addition to a low-level library.

Reply:
Any thoughts on this @ukstv